src/Sleep.js

```diff
@@ -1,26 +1,45 @@
 class Sleep {
-  constructor() {}
-  userSleepData(sleepData, id) {
-    this.userSleep = sleepData.filter(dailySleep => dailySleep.userID === id);
-    return this.userSleep;
+  constructor(sleepSet) {
+    this.sleepSet = sleepSet
   }
-  averageAllTimeSleep() {
-    let allTimeSleep = this.userSleep.reduce((sleep, day) =>{
-      sleep += day.sleepQuality;
-      return sleep 
+  userSleepData(id) {
+    return this.sleepSet.filter(dailySleep => dailySleep.userID === id);
+  }
```

Comment:
This method previously was not dynamic and would therefore only search for one user, this has been refactored to search based on a provided ID.

Reply:
looks cleaner then what we had 
I would like to see how this works in comparison